src/components/GridTable/Root.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box } from "@saleor/macaw-ui-next";
+import React, { TableHTMLAttributes } from "react";
+
+import { GridTableProps } from "./types";
+
+type GridTableRootElement = React.ElementRef<"div">;
+type GridTableRootProps = GridTableProps<TableHTMLAttributes<HTMLTableElement>>;
+
+export const GridTableRoot = React.forwardRef<GridTableRootElement, GridTableRootProps>(
+  (props, forwardedRef) => {
+    const { children, ...rest } = props;
+
+    return (
+      <Box height={0} {...rest}>
```

Comment:
suggestion: if you put all rest props here, we use the possibility to override the styles below, IMO should be spreaded on table